modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install

```diff
@@ -640,13 +640,22 @@ function social_group_flexible_group_update_11901(): string {
   return $updater->logger()->output();
 }
 
+/**
+ * Disable revisions for flexible group.
+ */
+function social_group_flexible_group_update_111102(): void {
+  $config = \Drupal::configFactory()->getEditable("group.type.flexible_group");
+  $config->set("new_revision", FALSE);
+  $config->save();
+}
+
 /**
  * Search for invalid permission(s) and remove them from existing roles.
  *
  * Permissions to check:
  * - "translate flexible_group group".
  */
-function social_group_flexible_group_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.